### DISCUSSION TITLE
Add placeholders for specific item progress to item missions

### DISCUSSION THREAD

**Discussion by Nekosity12** [BG-Software-LLC/SuperiorSkyblock2#282] [Pending Feature Requests] [answered]
So in example, blocks/kills missions both allow you to do {value_<block/entity>} in order to show the progress on that specific block/mob for multiple requirement quests. Item missions unfortunately don't have this capability and it would be nice as there are plenty of items you can't get to assign to stuff like block missions (I.E. apples/seeds/bones/rotten flesh). 

I know you have the item in your inventory so you should be able to view your own progress just by .. looking in your inventory. But it's a nice feature to have for erm "cosmetic" purposes as well as debugging. For example a quest we had an issue with in the past was for someone to collect to 64 oak planks and 16 oak saplings however I forgot to add the s to oak planks so it wouldn't register it. Since there's no additional placeholders on that mission I had to take a while to scan that mission and figure out what was wrong (Which not having an s didn't occur to me right away). Whereas if we had the placeholder I would have known the issue was instantly oak planks and could have focused solely on that to resolve the issue quicker.

I can't really see any negatives to adding this (besides having to put in the work to add it ofc :P) as even if someone doesn't want to have said placeholders... they don't have to use them ya know?

**Comment by Lemvs** [comment, marked as answer]
Implemented some time ago.

@OmerBenGera you can mark this answered and close it.
